### DISCUSSION TITLE
Elsa 3.1.3  [Activity( Category = "", DisplayName = "", Description = "", Outcomes = new[] { "True", "False" } )]In which namespace？

### DISCUSSION THREAD

**Discussion by shxylgithub** [elsa-workflows/elsa-core#5740] [General]
Elsa 3.1.3  [Activity(
Category = "",
DisplayName = "",
Description = "",
Outcomes = new[] { "True", "False" }
)]In which namespace？

**Comment by glime-ai** [comment]
In Elsa 3.1.3, the `[Activity]` attribute is used to define metadata for custom activities, including the category, display name, description, and outcomes. However, the namespace for the `[Activity]` attribute is not explicitly mentioned in the provided context.

In Elsa 2, attributes like `[Activity]` were typically found in the `Elsa.Attributes` namespace. Since Elsa 3 is a complete rewrite, there may have been changes to the namespaces and the way attributes are used. Given that the documentation and sources provided do not specify the namespace for the `[Activity]` attribute in Elsa 3, I would recommend checking the official Elsa 3 documentation or the source code for the correct namespace.

If the documentation is not yet complete, as is indicated in some of the context provided, you may need to explore the Elsa 3 source code on GitHub to find the namespace for the `[Activity]` attribute. Alternatively, you can reach out to the Elsa community or support channels for assistance.

Since I cannot provide a definitive answer without the current documentation or source code, please check the latest Elsa 3 documentation or source code for the most accurate information.

This comment was generated by [Glime](https://glimelab.ai/?utm_source=dRbvTb7IhCSYiI&utm_medium=github).
